metapool/sample_sheet.py

```diff
@@ -964,7 +974,7 @@ def make_sample_sheet(metadata, table, sequencer, lanes, strict=True):
         sequence'), forward and reverse barcode names ('i5 name', 'i7
         name'), description ('Sample'), well identifier ('Well'), project
         plate ('Project Plate'), project name ('Project Name'), and synthetic
-        DNA pool number ('synDNA pool number').
+        DNA pool number ('syndna_pool_number').
```

Comment:
Another benefit of string literals: only need to change these things in once place

Reply:
Noted